modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc

```diff
@@ -1,6 +1,8 @@
 [id="defining-the-list-of-images-to-pull_{context}"]
 = Defining the list of images to pull
 
+The {image-puller-name-short} can pre-pull most images, including scratch images (ex. che-machine-exec). However, images that mount volumes in the Dockerfile are not supported for pre-pulling on OpenShift.
```

Comment:
@ibuziuk for CRW `operator-metadata` is another example of a scratch image [1], and the `traefik` image mounts a volume in its Dockerfile [2], but they don't seem to be images that would be used for improving startup time. Do you think we should mention `operator-metadata` and `traefik` in the docs here?

[1] https://github.com/redhat-developer/codeready-workspaces-images/blob/f1af37bddf2f8aa7d16b0f9774eb4822edb30384/codeready-workspaces-operator-metadata/Dockerfile#L14
[2] https://github.com/redhat-developer/codeready-workspaces-images/blob/f1af37bddf2f8aa7d16b0f9774eb4822edb30384/codeready-workspaces-operator-metadata/Dockerfile#L14

Reply:
correct, it is non-sensical to pre-pull those non-workspace specific images e.g. operator-metadata / traefic - image puller is expected to pre-pull only images that are involved during workspace startup in order to speedup timing e.g. theia, brokers, sidecars etc